eZ/Bundle/EzPublishCoreBundle/Resources/config/templating.yml

```diff
@@ -45,6 +45,11 @@ parameters:
     ezpublish.view.configurator.class: eZ\Publish\Core\MVC\Symfony\View\Configurator\ViewProvider
     ezpublish.view_provider.registry.class: eZ\Publish\Core\MVC\Symfony\View\Provider\Registry
 
+    ezpublish.view_builder.content.default_templates:
+        full: content/full.html.twig
+        line: content/line.html.twig
+        embed: content/embed.html.twig
```

Comment:
And shouldn't we refer a path like `EzPublishCoreBundle:default:content/full.html.twig`?